### PR TITLE
Added a coverage report in the testing process

### PR DESCRIPTION
**Pull request by mssola**

### CODE REVIEW

**Comment by mssola** [comment]
It currently fails because this package does not abide to the current coverage threshold that we want for it.

**Comment by mssola** [comment]
Note also that the `before_script` is redundant because the `climate` script itself also tries to get the cover tool if it is not there, but I think it's better to write it down explicitly in the .travis.yml file itself

File: .travis.yml [hunk 1]
```diff
@@ -6,3 +6,9 @@ go:
 matrix:
   allow_failures:
     - go: tip
+
+before_script:
+  - go get golang.org/x/tools/cmd/cover
```
Comment (flavio) [inline .travis.yml]: AFAIK it changed to github.com/golang/tools/tree/master/cmd/cover

**Comment by flavio** [comment]
LGTM, next step: fix the code coverage ;)

**Comment by mssola** [comment]
So do I merge it now, even if it fails ? :D

**Comment by flavio** [comment]
Yes, as long as you start working on improving the code coverage

**Pull request merged**